src/cork/proposals.rs

```diff
@@ -220,3 +247,255 @@ pub async fn handle_scheduled_cork_proposal(
         }
     }
 }
+
+pub(crate) async fn handle_axelar_scheduled_cork_proposal(
```

Comment:
There's a very large amount of repeated code here, when the only differences are the proposal type, the chain ID parameter, and the call we need to make to schedule it. Is there a way we could use one code block and just set some values based on whether a proposal decodes as `ScheduledCork` or `AxelarScheduledCork`?